lib/bibliothecary/parsers/pypi.rb

```diff
@@ -252,18 +268,16 @@ def self.parse_requirements_txt(file_contents, options: {}) # rubocop:disable Li
             deps << result.merge(
               type: type
             )
-          else
-            match = line.delete(" ").match(REQUIREMENTS_REGEXP)
-            next unless match
-
+          elsif (match = line.delete(" ").match(REQUIREMENTS_REGEXP))
             deps << {
               name: match[1],
               requirement: match[-1] || "*",
               type: type,
             }
           end
```

Comment:
What things would fall into the `else` block? Should we test or alert on that condition?

Reply:
this change should be functionally the same (the `else` + `next` is basically just `elsif` here). I just decided to combine the 3 lines into 1 line randomly.